src/browser/ui/dom/CSSPropertyOperations.js

```diff
@@ -37,7 +37,11 @@ if (__DEV__) {
   // 'msTransform' is correct, but the other prefixes should be capitalized
   var badVendoredStyleNamePattern = /^(?:webkit|moz|o)[A-Z]/;
 
+  // style values shouldn't contain a semicolon
+  var badStyleValueWithSemicolonPattern = /;(\s*)$/;
```

Comment:
I think we can leave out the parens since we're not using that matching group anywhere. But good call on making the regex a bit more robust with the space checks.